README.md

```diff
@@ -22,17 +22,17 @@ We want our team to be a fun, productive, and safe space for all members. In add
 
 ## Expected behaviors
 
-Every member of the product team is expected to work hard, be considerate of their colleagues across the company, and contribute to a collaborative, positive, and healthy environment in which we can all succeed.
+Every member of the Engineering team is expected to work hard, be considerate of their colleagues across the company, and contribute to a collaborative, positive, and healthy environment in which we can all succeed.
 
 **Be supportive of your colleagues**, both proactively and responsively. Offer to help if you see someone struggling or otherwise in need of assistance (taking care not to be patronizing or disrespectful). If someone approaches you looking for help, be generous with your time; if you’re under a deadline, direct them to someone else who may be of assistance. Go out of your way to include people in team jokes or memes, recognizing that we want to build an environment free of cliques.
 
-**Be collaborative.** Involve your colleagues in brainstorms, sketching sessions, code reviews, planning documents, and the like. It’s not only okay to ask for help or feedback often, it’s unacceptable not to do so. Don’t succumb to either impostor syndrome (believing that you don’t deserve to be here) or [blowhard syndrome](http://xuhulk.tumblr.com/post/110549967516/stop-blowhard-syndrome) (believing you can do no wrong). Recognize that in addition to asking for feedback, you are similarly obligated to give it.
+**Be collaborative.** Involve your colleagues -- senior and junior -- in brainstorms, sketching sessions, code reviews, planning documents, and the like. It’s not only okay to ask for help or feedback often, it’s unacceptable not to do so. Don’t succumb to either impostor syndrome (believing that you don’t deserve to be here) or [blowhard syndrome](http://xuhulk.tumblr.com/post/110549967516/stop-blowhard-syndrome) (believing you can do no wrong). Recognize that in addition to asking for feedback, you are similarly obligated to give it.
 
-**Be generous and kind in both giving and accepting critique.** Critique is a natural and important part of our culture. Good critiques are kind, respectful, clear, and constructive, focused on goals and requirements rather than personal preferences. You are expected to give and receive criticism with grace.
+**Be generous and kind in both giving and accepting critique.** Critique is a natural and important part of our culture. Good critiques are kind, respectful, clear, and constructive, and focused on goals and requirements rather than personal preferences. You are expected to give and receive criticism with grace.
```

Comment:
@ecgeiser @danmactough I think this article is really, really great. The article that @gfogle posted is awesome, too. They are both so important that I suggest adding them as inline links, which gives them a lot more emphasis, rather then relegating them to the Further Reading, where they would be easy to skip over. 

For the article posted by Erica, we could add an additional sentence to the above paragraph that says "If a discussion becomes emotionally charged, try [using these techniques](https://hbr.org/2016/01/defusing-an-emotionally-charged-conversation-with-a-colleague) to defuse the situation and get the conversation back on track."